tensorflow/lite/experimental/litert/vendors/qualcomm/core/builders/rms_norm_op_builder.cc

```diff
@@ -28,37 +28,50 @@
 namespace qnn {
 
 static constexpr int kInputIndex = 0;
-static constexpr int kAxisIndex = 1;
+static constexpr int kGammaIndex = 1;
 
 std::vector<OpWrapper> BuildRmsNormOp(
     TensorPool& tensor_pool, const std::vector<TensorWrapperRef>& inputs,
     const std::vector<TensorWrapperRef>& outputs, const float epsilon) {
   std::vector<OpWrapper> res;
+
   auto& rms_norm_op = CreateOpWrapper(res, QNN_OP_RMS_NORM);
+  for (const auto& input : inputs) {
+    rms_norm_op.AddInputTensor(input);
+  }
 
   // Constructs axis param tensor.
   std::vector<std::uint32_t> axis_data;
-  axis_data.reserve(inputs[kAxisIndex].get().GetRank());
   axis_data.emplace_back(inputs[kInputIndex].get().GetRank() - 1);
   TensorWrapper& axis_tensor = tensor_pool.CreateStaticTensor(
       QNN_DATATYPE_UINT_32, inputs[kInputIndex].get().GetQuantParams(), {1},
       sizeof(std::uint32_t) * axis_data.size(), axis_data.data());
 
-  // Construct beta static all 0 tensor.
-  std::vector<int8_t> beta_data;
-  beta_data.reserve(GetDataTypeSize(inputs[kAxisIndex].get().GetDataType()) *
-                    inputs[kAxisIndex].get().GetTensorSize());
-  std::memset(beta_data.data(), 0, beta_data.size());
-  TensorWrapper& beta_tensor = tensor_pool.CreateStaticTensor(
-      inputs[kAxisIndex].get().GetDataType(),
-      inputs[kAxisIndex].get().GetQuantParams(),
-      inputs[kAxisIndex].get().GetDims(), sizeof(int8_t) * beta_data.size(),
-      beta_data.data());
+  if (inputs[kGammaIndex].get().GetDataType() == QNN_DATATYPE_FLOAT_32) {
+    // Construct float beta static all 0 tensor.
+    std::vector<float> beta_data;
+    beta_data.reserve(inputs[kGammaIndex].get().GetTensorNumElements());
+    std::memset(beta_data.data(), 0, beta_data.size());
```

Comment:
Prefer using the member function of std::vector to std::memset.

```suggestion
    std::vector<float> beta_data(inputs[kGammaIndex].get().GetTensorNumElements(), 0);
```